templates/types/streaming/nextjs/app/components/ui/chat/chat-message/codeblock.tsx

```diff
@@ -1,19 +1,16 @@
 "use client";
 
+import hljs from "highlight.js";
+import "highlight.js/styles/atom-one-dark-reasonable.css";
```

Comment:
```suggestion
// instead of atom-one-dark theme, there are a lot of others: https://highlightjs.org/demo
import "highlight.js/styles/atom-one-dark-reasonable.css";
```